src/components/annotate/sidePanel/AnnotationForm.tsx

```diff
@@ -102,14 +120,16 @@ const AnnotationForm = (props: Props) => {
 
             /* Set Class form fields */
             setClassFormFields(classForm.formFields);
+        } catch {
+            annotationValue = !isEmpty(editAnnotation) ? editAnnotation['oa:body']['oa:value'] : get(annotateTarget.target, targetField) as unknown[];
```

Comment:
Same as the above.

Reply:
Also the same explanation

Reply:
Removed try catch, and replaced with if else statement